src/JuMP.jl

```diff
@@ -954,6 +954,15 @@ function Base.setindex!(m::JuMP.Model, value, name::Symbol)
     m.obj_dict[name] = value
 end
 
+"""
+    Base.haskey(m::JuMP.AbstractModel, name::Symbol)
```

Comment:
remove `Base.`

Reply:
m -> model

Reply:
From just the docstring or the code also?

Reply:
only from docstrings